ducktape/cluster/json.py

```diff
@@ -12,9 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from ducktape.command_line.config import ConsoleConfig
```

Comment:
I think you need `from __future__ import absolute_import` here as well. 

When I tested with `JsonCluster` on my local machine, it failed to load the json file because it was picking up the wrong `json` module. However using absolute_import fixed this